l3build-manifest.lua

```diff
@@ -33,6 +33,9 @@ for those people who are interested.
       `l3build-manifest-setup.lua`.
 --]]
 
+local ctanfiles
+local tdsfiles
```

Comment:
Shouldn't this be local insdie the function, so e.g. `local ctanfiles = {}`?

Reply:
No because it is shared by different functions of this module